### PR TITLE
add tempo ingest scripts

### PR DESCRIPTION
**Pull request by jeromebarre**
## Description

add tempo ingest scripts

## Issue(s) addressed

https://github.com/JCSDA-internal/skylab/issues/111

## Dependencies

none

## Impact

none

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have run the unit tests before creating the PR

### CODE REVIEW

File: scripts/nasa/obs/tempo/store_obs.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+from r2d2 import R2D2Data
+from solo.configuration import Configuration
+from solo.date import date_sequence, JediDate, DateIncrement
+
+
+def store_observations(yaml_file):
+    config = Configuration(yaml_file)
+    dates = date_sequence(config.start, config.end, config.window_length)
+    provider = config.provider
+    #type = config.type
```
Comment (fabiolrdiniz) [inline scripts/nasa/obs/tempo/store_obs.py:13]:
```suggestion
```

File: scripts/nasa/obs/tempo/store_obs.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+    for date in dates:
+        date = JediDate(date)
+        window_start = date - DateIncrement('PT3H')
+        source_file=f'{source_path}/TEMPO_Intelsat40e_{date.year()}{date.month()}{date.day()}{date.hour()}.nc'
+        if not os.path.isfile(source_file): source_file=f'{source_path}/empty.nc' 
+        print(source_file)
+        R2D2Data.store(
+             provider=provider,
+             #type=type,
```
Comment (fabiolrdiniz) [inline scripts/nasa/obs/tempo/store_obs.py:27]:
```suggestion
```

File: scripts/nasa/obs/tempo/config_store_obs.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+start: 2021-08-02T00:00:00Z
+end: 2021-08-03T00:00:00Z
+provider: nasa
+type: obs
```
Comment (fabiolrdiniz) [inline scripts/nasa/obs/tempo/config_store_obs.yaml:4]:
```suggestion
```

File: scripts/nasa/obs/tempo/store_obs.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    item =  config.item
+    window_length = config.window_length
+    source_path = config.source_path
+    observation_type = config.obs_type
+
+    for date in dates:
+        date = JediDate(date)
+        window_start = date - DateIncrement('PT3H')
+        source_file=f'{source_path}/TEMPO_Intelsat40e_{date.year()}{date.month()}{date.day()}{date.hour()}.nc'
+        if not os.path.isfile(source_file): source_file=f'{source_path}/empty.nc'
```
Comment (fabiolrdiniz) [inline scripts/nasa/obs/tempo/store_obs.py:23]: I know it adds a line, but it's a little hard to keep track at first look. I believe that `pylint` complains about it too.
```suggestion
        if not os.path.isfile(source_file):
            source_file=f'{source_path}/empty.nc' 
```

File: scripts/nasa/obs/tempo/store_obs.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    window_length = config.window_length
+    source_path = config.source_path
+    observation_type = config.obs_type
+
+    for date in dates:
+        date = JediDate(date)
+        window_start = date - DateIncrement('PT3H')
+        source_file=f'{source_path}/TEMPO_Intelsat40e_{date.year()}{date.month()}{date.day()}{date.hour()}.nc'
+        if not os.path.isfile(source_file): source_file=f'{source_path}/empty.nc' 
+        print(source_file)
```
Comment (fabiolrdiniz) [inline scripts/nasa/obs/tempo/store_obs.py:24]: R2D2 will print out the file's name depending on the log level.
```suggestion
```

**Review by fabiolrdiniz — APPROVED** [review]
Thanks, @jeromebarre! I believe it's safe to remove the `type` since that's deprecated.

File: scripts/nasa/obs/tempo/store_obs.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    item =  config.item
+    window_length = config.window_length
+    source_path = config.source_path
+    observation_type = config.obs_type
+
+    for date in dates:
+        date = JediDate(date)
+        window_start = date - DateIncrement('PT3H')
+        source_file=f'{source_path}/TEMPO_Intelsat40e_{date.year()}{date.month()}{date.day()}{date.hour()}.nc'
+        if not os.path.isfile(source_file):
```
Comment (mer-a-o) [inline scripts/nasa/obs/tempo/store_obs.py:22]: The code looks good. Just one suggestion, print a warning/message when the file is not available and `empty.nc` is used.

**Review by mer-a-o — APPROVED** [review]

**Review by ericlingerfelt — APPROVED** [review]
Thanks, @jeromebarre!

**Pull request merged**